backend/tests/dealershipControllerTest.js

```diff
@@ -10,9 +10,31 @@ chai.use(chaiHttp);
 
 let dealership_id = "";
 
-const token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2MThhYWNmNDVjZGM3NWI4Mjg4ZWI5YjUiLCJpYXQiOjE2MzgxMzc1ODUsImV4cCI6MTY0MDcyOTU4NX0.gv4Q-vPm-MqfHVsY7BYnBThxBI3bHZtUC5JtukHp340';
+//creds
+const email = "admin@account.com";
+const password = "password";
```

Comment:
Exposed credentials is never a good thing even though right now it doesn't really matter. We should find a fix for this and change it for the rest of the tests

Reply:
Yes, I agree. However, ideally we should save those creds in a config file that will never be pushed to github. However, when jenkins is set up to run those tests it will need to have access to that config file. Thus, we need to set up a docker image and save those creds in the docker image. Will do this update once the docker image is up and running.

Reply:
Ok, in the meanwhile we will have to manually review the SonarCloud security issues until this is fixed.